src/components/SummaryTable/index.jsx

```diff
@@ -1,30 +1,42 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import StatusWidget from '../StatusWidget';
+import CONFIG from '../../utils/nimbledroid/config';
+import { frum } from '../../utils/queryOps';
```

Comment:
It is meant to reference the `FROM` clause in SQL.  I can think of no other name. I am open to suggestions.

Reply:
When it comes to naming, I find it helpful to think of describing what a function does (or what a variable is/contains), even if it seems overly simplistic or obvious. In my opinion, the more descriptive, the more readable -  and the less need for explanatory comments. So `frum` could be renamed to `newInstanceFrom` or `newWrapperFrom`. Though to be honest, you need this method since all its doing is adding a layer of syntax to `new Wrapper` and not adding any customized behavior :)